### DISCUSSION TITLE
نسخه ایفون

### DISCUSSION THREAD

**Discussion by aryanveg** [hiddify/hiddify-app#952] [General]
سلام وقت بخیر
برای استفاده در گوشی های اپل از کدوم نسخه استفاده کنیم؟

**Comment by mtmoein** [comment]
سلام من در آیفن 13 ورژن 17.4.1 تمام نسخه ها را تست کردم ولی جواب نگرفتم.
توسعه برای آیفن واقعا ضعیف بوده.  :(
